Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/PokePointer.cs

```diff
@@ -51,6 +52,25 @@ public class PokePointer : BaseControllerPointer, IMixedRealityNearPointer
         /// </summary>
         public int SceneQueryBufferSize => sceneQueryBufferSize;
 
+        [SerializeField]
+        [Tooltip("Whether to ignore colliders that may be near the pointer, but not actually in the visual FOV. " +
+            "This can prevent accidental touches, and will allow hand rays to turn on when you may be near a " +
+            "touchable but cannot see it. Visual FOV is defined by cone centered about display center, " +
+            "radius equal to half display height.")]
+        private bool ignoreCollidersNotInFOV = true;
+       
+        /// <summary>
+        /// Whether to ignore colliders that may be near the pointer, but not actually in the visual FOV.
+        /// This can prevent accidental touches, and will allow hand rays to turn on when you may be near 
+        /// a touchable but cannot see it. Visual FOV is defined by cone centered about display center, 
+        /// radius equal to half display height.
+        /// </summary>
+        public bool IgnoreCollidersNotInFOV
```

Comment:
Maybe? I'm hesitant to modify interfaces since it's hard to undo or change interfaces.

Reply:
I agree with Julia, feels more like feature creep for the IMixedRealityPointer interface. 

But it does seem like this would be good in a base class for PokePointer and SpherePointer

Reply:
I would prefer not to add more levels of inheritance until there's lots of signal (3 repeated classes) that a field could be re-used. Motivation is I find it really hard to navigate the code in MRTK because many fields are in many levels of base classes. Maybe I just need a better tool for navigating base classes (currently use class diagrams). I am thinking of making a third kind of near pointer, a Cone Pointer, as part of fixing #6634 (which I will probalby break up into multiple bugs). Then it woudl be appropriate to have a base class, I think.

Reply:
Very true, base classes impede performance and sometimes code maintainability.

Alternative suggestion...to just consider...because I myself am not sure it's better/good. Why not create a new interface, like IFOVChecker? 

Just a thought

Reply:
I'd prefer to reduce the number of new interfaces we add. I think when I build the cone pointer I will probably make a base class or interface, but for now I'll leave it like this...keeps the code simpler